docs/distribution/README.md

```diff
@@ -0,0 +1,923 @@
+# OCI Distribution - Notary v2 Signature Support
+
+To support [Notary v2 goals][notaryv2-goals], upload, persistance and discovery of signatures must be supported.
+
+To minimize the complexity of registry operators and projects to adopt Notary v2, a balance between leveraging what already exists and new patterns to support secure discovery are explored.
+
+## Table of Contents
+
+* [Signature Persistance](#signature-persistance)
+* [Signature Push](#signature-push)
+* [Signature Discovery](#signature-discovery)
+* [Signature Pull](#signature-pull)
+* [Example Artifacts](#example-artifacts)
+
+## Signature Persistance
+
+Several options for how to persist a signature were explored. We measure these options against the [goals of Notary v2][notaryv2-goals], specifically:
+
+* Maintain the original artifact digest and collection of associated tags, supporting existing dev through deployment workflows
+* Multiple signatures per artifact, enabling the originating vendor signature, public registry certification and user/environment signatures
```

Comment:
So potentially allowing for recursive signatures = Sign(Payload, Annotations, Sign(Payload, Annotationss, Sign(...))) as the artifact gets moved around?

Reply:
There was a recent conversation about signing signatures in the 3T SBOM working group as well. It was thought to be more complicated and not clear if it's needed. The current model we're focusing on is independent, parallel signatures. In our core example, acme-rockets may decide to only add their signature if the wabbit-networks signature first exists, and is valid. But the acme-rockets signature would be persisted independently of the wabbit-networks signature. This would mean the production environment only needs to validate the existence of the acme-rockets signature. 
They could also say they must have both the acme-rockets signature and wabbit-networks signature as part of their policy management. But, they're not coupled, today.

Can you think of a scenario where we need chained signatures?